tar/chunks/chunks_stats_internal.c

```diff
@@ -63,6 +63,21 @@ chunks_stats_addstats(struct chunkstats * to, struct chunkstats * from)
 	to->s_zlen += from->s_zlen;
 }
 
+/* For use with chunks_stats_printheader() and chunks_stats_print(). */
+#define print_field(format, value, err) do {				\
+	if (fprintf(stream, (format), (value)) < 0) {			\
+		warnp("fprintf");					\
+		goto err;						\
```

Comment:
I know that we'd normally put parentheses around macro arguments, but `goto (err);` didn't work.

I tried writing `goto err ## err_num;` and passing in `0` or `2` instead of `err0` or `err2`, but then the macro invocation was harder to understand.